README.md

```diff
@@ -64,9 +64,10 @@ $ node-lambda run --help
 
   Options:
 
-    -h, --help                     output usage information
-    -h, --handler [index.handler]  Lambda Handler {index.handler}
-    -j, --eventFile [event.json]   Event JSON File
+    -h, --help                          output usage information
+    -h, --handler [index.handler]       Lambda Handler {index.handler}
+    -j, --eventFile [event.json]        Event JSON File
+    -x, --contextFile [context.json]    Context JSON file
```

Comment:
I think using `-x` for excludes makes a bit more sense:
https://github.com/motdotla/node-lambda/pull/48
Perhaps using `-c` makes sense here?